mava/adders/reverb/sequence.py

```diff
@@ -51,6 +54,8 @@ def __init__(
         client: reverb.Client,
         sequence_length: int,
         period: int,
+        int_to_nets: List[str] = None,
```

Comment:
What does `int_to_nets` mean? Also, please update the docstrings wherever appropriate.

Reply:
I updated the docstrings of the system.py and adders. int_to_nets is used to convert integers to network keys. The reason this is used is the networks_keys need to be sent to the adders. This is to allow the adder to know to which table to write parts of each data. The problem is that the adders don't like to work with strings, because at some point zero padding is used. So now I use ints which maps to the network keys.

Reply:
Should I maybe call `int_to_nets` > `int_ids_to_net_keys_map` ?

Reply:
Can we do this right before adding. Like having everything like before, just before adding convert to int?